src/components/SignupForm/SignupForm.module.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/colors';
+
+.signup-form {
+  background-color: $white-main;
+  border-top: 6px solid $blue;
+  max-width: 300px;
+  margin: auto;
```

Comment:
alphabetical order.

Also if you are positioning elements, use FLEX, not text-align pls